terraform/modules/opensearch_domain/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "instance_type" {
```

Comment:
We should define a `type` on all of our variables so Terraform can validate the values that we pass in